packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -264,6 +264,8 @@ let spawnedWorkDuringRender: null | Array<ExpirationTime> = null;
 // receive the same expiration time. Otherwise we get tearing.
 let currentEventTime: ExpirationTime = NoWork;
 
+const SuspenseTimeouts: Map<TimeoutID, FiberRoot> = new Map();
```

Comment:
This isn't gated behind a feature flag, or behind a DEV-check. Seems the only reason is for backwards compat with React DOM. But legacy code all runs in sync mode.

So how about we only enable this in the testing build? The normal build will warn if you're in concurrent mode and you're wrapped in `act` and you suspend.

Reply:
yeah I'm thinking that's a good idea too. this is ideal for being only in a testing build.